cue/schemas/panels/table/migrate.cue

```diff
@@ -14,27 +14,41 @@ if (*#panel.type | null) == "table" {
 			#var: string
 			output: [
 				// Rename anonymous fields that Perses names differently than Grafana
-				if #var == "Time" { "timestamp" },
-				if #var == "Value" { "value" },
-				#var
+				if #var == "Time" {"timestamp"},
+				if #var == "Value" {"value"},
+				#var,
 			][0]
 		}
 
 		columnSettings: list.Concat([
 			for transformation in (*#panel.transformations | [])
-				if transformation.id == "organize" {
-					list.Concat([
-						[for columnName, displayName in (*transformation.options.renameByName | {}) {
-							name: {_nameBuilder & {#var: columnName}}.output
-							header: displayName
-						}],
-						[for columnName, isExcluded in (*transformation.options.excludeByName | {}) {
-							name: {_nameBuilder & {#var: columnName}}.output
-							hide: isExcluded
-						}]
-					])
-				}
-			,
+			if transformation.id == "organize" && len((*transformation.options.indexByName | {})) == 0 {
```

Comment:
maybe a small comment to explain why we go through the transforms 2 times here?
I wonder if this can be improved.. but let's keep that for another PR 😅